### DISCUSSION TITLE
How do playlists work ?

### DISCUSSION THREAD

**Discussion by Revhanator** [vzhd1701/gridplayer#144] [Q&A] [answered]
Hi everyone !
I'm trying to figure out how do playlists work. When I add all my video files to the playlist, it actually opens all of them at once (and crashes since there are so many) instead of VLC usual behavior of playing the files one after the other from the playlist.

What I'd like to achieve is having a 2x2 grid or 3x3 grid (depending on settings) that would play random files from my playlist in each cell.

Any help is welcome !
Rev

**Comment by vzhd1701** [comment, marked as answer]
This behavior is not supported yet. Currently all videos in the playlist are displayed. You can add videos from one directory, enable `Playback -> Loop -> Repeat Directory (Shuffle)` and it will play random videos while keeping the grid configuration.

**Reply by Revhanator** [reply]
Thanks for your reply, it was very helpful in getting it to work very close to what I described !
Rev
